defaults/main.yml

```diff
@@ -52,6 +52,8 @@ archivematica_src_am_wait_local_mysql: "true"    # MCPServer upstart service to
 # Environment strings
 #
 
+# archivematica_src_ca_custom_bundle can be defined as var. If not defined it will be autogenerated by the playbook and AM will use the OS CA bundle. On Ubuntu, the system ca bundle /etc/ssl/certs/ca-certificates.crt file will be used. The /etc/pki/tls/certs/ca-bundle.crt file is the system ca bundle on CentOS. For more info, refer to the CA certificates at Security section in the Archivematica Documentation.
```

Comment:
I think that you missed `archivematica_src_am_mcpserver_environment`?
Is that because it doesn't install `requests`? That would make sense.

Reply:
@mamedin can you add a link to artefactual-labs/am-packbuild#140 for context?